eox_tenant/models.py

```diff
@@ -112,7 +112,7 @@ def get_value_for_org(cls, org, val_name, current_value):
             if value is None:
                 continue
 
-            if current_value == value:
+            if result.key == current_tenant:
```

Comment:
@navinkarkera Should we also check here if teh `EDNX_USE_SIGNAL` for this tenant config is set to true ?

Reply:
It should not be required as the only place this function is called is in proxy which is already verifying the flag. Now even if the `current_tenant` is set to None (is `EDNX_USE_SIGNAL` is false), this condition will never pass and the function will just return first valid org value.